packages/zowe-explorer-api/src/profiles/ZoweExplorerApi.ts

```diff
@@ -345,9 +345,18 @@ export namespace ZoweExplorerApi {
      * @export
      */
     export interface IJes extends ICommon {
+        /**
+         * Returns a list of jobs for any parameters.
+         *
+         * @param {string} owner
+         * @returns {Promise<zowe.IJob[]>} an array if IJob
+         */
+        getJobsByParameters?(params: zowe.IGetJobsParms): Promise<zowe.IJob[]>;
+
         /**
          * Returns a list of jobs for a specific user and prefix.
          *
+         * @deprecated Use getJobsByOwnerAndPrefix
```

Comment:
Not sure if this deprecation warning belongs here, since it suggests to use this same deprecated function. Or is there another `getJobsByOwnerAndPrefix` that it should reference instead?

Reply:
Good catch, I fixed to reference the new function.